Contribution.md

```diff
@@ -1,16 +1,21 @@
 # How to get started
 
-In order to run the SharpBuckets integration tests you should create OAuth tokens for your Bitbucket account. It is recommended that you create a separate Bitbucket account for developing.
+In order to run the SharpBucket's integration tests you should configure your development environment to inject OAuth credentials that will be used to connect to Bitbucket.  
+We recommend you to create a separate Bitbucket account that will be used only to run the tests. (disconnect and go [here](https://bitbucket.org/account/signup/) to create your test account)  
```

Comment:
Maybe log out is better than disconnect.

Reply:
right, I do the change